### ISSUE TITLE
Move Expression Language TCK workflow to new EL baseline

### ISSUE THREAD

**Issue by mnriem** [piranhacloud/piranha#2610]

**Comment by mnriem** [comment]
Blocked on Expressly 5.0.0 release

**Comment by mnriem** [comment]
Tracking upgrade to Expressly 5.0.0 from M2 milestone as a separate issue.
